.coderabbit.yaml

```
@@ -38,7 +41,9 @@ reviews:
       instructions: |
         This folder, including "main" files, are all for the purpose of testing. This is because an annotation processor can only really be tested by applying it to a build
 
-        Coding standards can be relaxed a little here, as the code isn't published and isn't used by anyone except for tests
+        Coding standards can be relaxed a little here, as the code isn't published and isn't used by anyone except for tests. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)